MyPerfectOnboardingApplication/Tests/MyPerfectOnboarding.App.Tests/Api/Controllers/ListControllerApplicationTests.cs

```diff
@@ -0,0 +1,282 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using MyPerfectOnboarding.Api.Controllers;
+using MyPerfectOnboarding.Contracts.Database;
+using MyPerfectOnboarding.Contracts.Models;
+using MyPerfectOnboarding.Contracts.Services.Generators;
+using MyPerfectOnboarding.Contracts.Services.Location;
+using MyPerfectOnboarding.Dependency;
+using MyPerfectOnboarding.Dependency.Containers;
+using MyPerfectOnboarding.Tests.Utils.Builders;
+using MyPerfectOnboarding.Tests.Utils.Comparers;
+using MyPerfectOnboarding.Tests.Utils.Extensions;
+using NSubstitute;
+using NUnit.Framework;
+using Unity;
+
+namespace MyPerfectOnboarding.App.Tests.Api.Controllers
```

Comment:
```suggestion
namespace MyPerfectOnboarding.Application.Tests.Api.Controllers
```
https://github.com/HanaNavratilova/kentico-onboarding-cs/pull/4#discussion_r261535994 💩